onnx/compose.py

```diff
@@ -420,7 +420,9 @@ def add_prefix_graph(
     rename_outputs: Optional[bool] = True,
     rename_initializers: Optional[bool] = True,
     rename_value_infos: Optional[bool] = True,
+    rename_attribute_graph: Optional[bool] = True,
```

Comment:
Can we drop this flag? I am not sure if it makes sense to set some other rename flag to true, and make this false. We may as well recursively process subgraphs always.

Reply:
Good point. Just removed. Thanks!